packages/zowe-explorer/README.md

```diff
@@ -1,160 +1,161 @@
+6. Edit the configuration file to include the host and other connection information for a mainframe service, and save.
 
-Your team configuration file appears either in your .zowe folder if you chose the global configuration file option, or in your workspace directory if you chose the project-level configuration file option. The notification message that shows in VS Code after config file creation includes the path of the created file.
+      Your team configuration file appears either in your `.zowe` folder if you chose the global configuration file option, or in your workspace directory if you chose the project-level configuration file option. The notification message that displays in VS Code after configuration file creation includes the path of the created file.
 
-You can now use all the functionalities of the extension.
+      You can now use all the functionalities of the extension.
 
 ### Using profiles for the first time
 
-The first time profiles are used you will be prompted for user name and password for the profile's connection. The term password is used loosely to represent all supported authentication secrets like passphrases, passtickets, Multifactor Authentication (MFA) tokens, etc.
+The first time team configuration profiles are used, you are prompted for a user name and password for the profile's connection.
 
-### Updating securely stored credentials
-
-Secure fields in the team configuration file are handled by the Zowe Imperative dependency. To update securely stored user names and passwords in Zowe Explorer, the user can right click the profile and select **Manage Profile**, then **Update Credentials** from the drop down list. This prompts the user for the new credentials and the secure credentials vault is updated.
+The term *password* is used loosely to represent all supported authentication secrets, such as passphrases, passtickets, Multifactor Authentication (MFA) tokens, etc.
 
-### Editing team configuration file
+### Profile validation
 
-1. Navigate to the **Side Bar**.
-1. Hover over **DATA SETS**, **USS**, or **JOBS**.
-1. Click the **+** icon.
-1. If team configuration file is in place, the **Edit Team Configuration File** option displays.
-   ![Edit Team Configuration File](/docs/images/ZE-edit-config.png)
-   <br /><br />
-1. If only a global or project level config is in place, it opens to be edited. If both a global and project level config are in place, the user must select which file to edit.
-   ![Edit Config Location Option](/docs/images/ZE-edit-options.png)
-   <br /><br />
+Zowe Explorer for VS Code includes a back-end profile validation feature that helps to ensure that the specified connection to z/OS is successfully established and your profile is ready for use. If the API connection is valid, the profile is active and can be used.
```

Comment:
No strong preference, but "back-end" could be removed from this sentence

Reply:
Understand what you're saying. But I feel like including "back-end" is good here to explain that there is no user action needed for this to happen, nor is there any visual confirmation if the validation passes.